libs/components/lookup/src/lib/modules/lookup/lookup-show-more-modal.component.ts

```diff
@@ -136,8 +136,13 @@ export class SkyLookupShowMoreModalComponent
       this.#changeDetector.markForCheck();
     }
 
+    // if onlyShowSelected is checked, then only iterate through the initial selection
+    const items = this.items.filter(
```

Comment:
I wonder if doing the filter conditionally when `this.onlyShowSelected` is true is a little more efficient. With the way this is written we will iterate through all of `this.items` every time no matter if it is needed or not. Some consumers load a crazy amount of data into `lookup` so I wonder if we should only do the `.filter` if it is needed and just set the local variable to `this.items` if not.

Reply:
Good idea. Changed